boards/arm/relax_kit/relax_kit_defconfig

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Since the CPU has ARM MPU, can you enable by default the ARM_MPU here? See how it is done in other defconfig of ARM boards.

Reply:
I have a compilation failure when enabled. Sure I will check this, thanks.

Reply:
@parthitce  i think you need to include devicetree.h in your soc.h.

```
/* Add include for DTS generated information */
#include <devicetree.h>

```
 See, for example the Nordic SOCS.

Let me know if this solves your error

Reply:
@ioannisg Yes, that solves the problem. Thank you.